src/runtime_src/core/include/xrt_mem.h

```diff
@@ -70,7 +70,7 @@ struct xcl_bo_flags
  * bits  0 ~ 15: DDR BANK index
  * bits 24 ~ 31: BO flags
  */
-#define XRT_BO_FLAGS_MEMIDX_MASK	(0xFFFFFFUL)
+#define XRT_BO_FLAGS_MEMIDX_MASK	(0xFFFFUL)
```

Comment:
Wow, how did this ever work?

Reply:
This changes doesn't have slot information. Hence this will not effect anything.